static/js/map.js

```diff
@@ -0,0 +1,251 @@
+// const modisURL = "modis.csv";
+const modisURL = "https://australia-fire-api-dashboard.herokuapp.com/api/v1.0/fires_modis";
+const viirsURL = "https://australia-fire-api-dashboard.herokuapp.com/api/v1.0/fires_viirs";
+const states = "https://raw.githubusercontent.com/rowanhogan/australian-states/master/states.geojson";
+const areaURL = "http://australia-fire-api-dashboard.herokuapp.com/api/v1.0/fires_bystate";
+
+// TODO load in second dataset, create function that allows them to choose which dataset to use
+
+//////////// IMPORT THE DATA //////////////////
+function getData(modisURL, viirsURL) {
+  d3.json(modisURL).then(modisData => {
+
+    d3.json(viirsURL).then(viirsData => {
+
+      d3.json(areaURL).then(areaData => {
+
+        d3.json(states).then(stateData => {
+
+          let borderData = stateData.features;
+          
+          makeFeatures(modisData, viirsData, borderData, areaData);
+    
+        }).catch(e => {
+          console.log(e);
+        });
+
+      });      
+      
+
+    }).catch(e => {
+      console.log(e);
+
+    });
+      
+  }).catch(e => {
+    console.log(e);
+  });
+}
+
+getData(modisURL, viirsURL);
+
+//////////// CALCULATE ANIMAL DEATHS PER STATE //////////////////
+// Uses general formula described in https://www.bbc.com/news/50986293
+
+// variable to hold calculations, to make later popover
+const deathsByState = [];
+
+function animalDeaths(hectareData, stateName) {
```

Comment:
This is awesome! I'm slightly worried about performance doing this calculation on the frontend in the browser vs doing this on the backend in an endpoint. But, we can leave it and test it against the whole dataset later.

No need to make any changes now... just something to take note...